run.py

```
@@ -8,8 +8,8 @@
 from app.config import settings
 
 if __name__ == "__main__":
-    assert settings.LENS_URL, "Missing LENS_URL from environment"  
-    assert settings.LENS_TOKEN, "Missing LENS_TOKEN from environment"
+    assert settings.lens_url, "Missing LENS_URL from environment"  
```

Ruff (0.12.2)

Use of `assert` detected

(S101)